reqs/requirements-dev-contrib.txt

```diff
@@ -4,7 +4,7 @@ invoke>=1.7.1
 isort==5.10.1
 mypy==0.991
 pre-commit>=2.6.0
-pydantic>=1.10
+pydantic>=1.0<2.0 # needed for mypy plugin support
```

Comment:
@Kilo59 Is it appropriate to have the same line in multiple files?  Or should we consider refactoring?  Perhaps in a follow-on pull request?  Thanks.